students/Hebdowska_Grazyna/lesson_06/access_right.py

```diff
@@ -0,0 +1,50 @@
+def input_files():
+    n = int(input())
+    new_file = {}
+    new_file["X"] = []
+    new_file["R"] = []
+    new_file["W"] = []
+    for i in range(n):
+        nm = input()
+        nm = nm.split(' ')
+        if "X" in nm:
+            new_file["X"].append(nm[0])
+        if "R" in nm:
+            new_file["R"].append(nm[0])
+        if "W" in nm:
+            new_file["W"].append(nm[0])
+    return new_file
+
+
+def input_request():
+    n = int(input())
+    new_file = []
+    for i in range(n):
+        nm = input()
+        nm = nm.split(' ')
+        new_file.append(nm)
+    return new_file
+
+
+def result(a, b):
+    for i in b:
+        if i[0] == "write":
```

Comment:
Please try to avoid magic numbers like 0,1
```
if i[0] == "write":
if i[1] in a["W"]:
```

You might create defines for it e.g
```
FILE_OPERATION = 0
FILE_PERMISSION = 1
if i[FILE_OPERATION ] == "write":
...
if i[FILE_PERMISSION ] in a["W"]:
```